content/docs/guides/multi-target-exporter.md

```diff
@@ -315,10 +316,11 @@ scrape_configs:
 `prometheus.yml` for macOS and Windows:
 
 ```yaml
-scrape_configs:
-
-- job_name: blackbox-exporter-metrics
+global:
   scrape_interval: 5s
+
+scrape_configs:
+- job_name: blackbox-exporter # to get metrics about the exporter itself
```

Comment:
It might be better to put these above rather than beside

Reply:
It is already not very good to even have a comment instead of a more explicit name. If we move it up the connection will be completely lost.

Reply:
Having a comment before the block it refers to is pretty normal, I'm more thinking about the horizontal space this takes up.

Reply:
This 72 chars (with spaces) in total. Small enough for an old terminal.

And "normal" was not my argument. I want to teach something. I needs to be there so people can make the connection when only focusing on one line. Two lines are far more complicated and won’t work as much.